_includes/utilities/js/helpers/wrapStringPhrases.js

```diff
@@ -1,15 +1,26 @@
-export default function wrapStringPhrases(string, phrases, templateFn) {
+export default function wrapStringPhrases(string, phrasesArray, templateFn, options = {}) {
```

Comment:
what does it mean to wrapStringPhrases? I can't tell from the code so perhaps a comment is in order.

Reply:
@jbothma Would adding http://usejsdoc.org/ annotation to my functions be helpful?

Reply:
depends. Just a comment would help. I haven't gone around looking for all the cases where you use this to understand what purpose it serves. It doesn't need fancy annotated docs, it just needs an explanation of what this function is for. If your code is functional, you can say this takes x as input and gives y as output, right?

Reply:
Right, this makes sense.